language/stdlib/modules/move_version.move

```diff
@@ -0,0 +1,32 @@
+address 0x0:
+
+module MoveVersion {
+    use 0x0::LibraConfig;
+    use 0x0::Transaction;
+
+    struct T {
+        major: u8,
+        minor: u8,
+    }
+
+    public fun initialize() {
+
```

Comment:
Delete?

Reply:
We need to set it up at genesis.  Maybe change it to private?

Reply:
Ah, I just meant that there's an extra newline here.